README.md

```diff
@@ -1 +1,87 @@
-# LF_test_task
+<h1> Test task service API </h1>
+
+<h3> REST API version </h3>
+The most recent version of the API is v1.
+
+<h2> Running Docker image <i>(Linux)</i></h2>
+
+In order to run this container you'll need docker [installed](https://docs.docker.com/install/)
+
+In the `project` directory run
+
+```shell
+$ sudo docker build -t my_docker_flask:latest .
+$ sudo docker run -d -p 5000:5000 my_docker_flask:latest
+```
+
+Check if container is running by going to [http://127.0.0.1:5000/](http://127.0.0.1:5000/)
+
+<h2> Available endpoints and how to use them </h2>
+
+<h3> Method 1: Get an average elapsed time for all queries on certain date </h3>
+
+<b>Description:</b> This endpoint returns an average elapsed time for all queries on certain date 
+
+<b>Request URL:</b> http://127.0.0.1:5000/api/v1/queries/avg_elapsed_time?date=[:date]
```

Comment:
Though it is clear that method is `GET` here, it is always appropriate to add method type into your API specification (`GET`, `PUT`, `POST`, `DELETE`).